proxima-chapter-1.md

```diff
@@ -26,17 +25,18 @@ The dialog with the smart city was implemented through a number of channels: Cha
+
+Then, Covid-19 begun to hit our lives. The Smart City concept morphed into something different: as a citizen I'd be less interested in knowing that the subway or my bus ride has a 4.2 minutes delay compared to the original schedule, and possibly be mindful in determining if I *can* take advantage of different urban services in a *safe* fashion, extending the safety protocols to closed environments (offices, university rooms, workplaces). And, matter of fact, this is the idea we'll be facing here, trying to build a lab as a tool to design safety-related use cases.
 
 *Mutatis Mutandis*, we'll modify the original patterns and configurations exposed with **Proxima City** to allow a Smart City Lab become a Safe City Lab, hence the name ProximaSafe. We'll explore a number of technologies involved in different areas - with Oracle Cloud Infrastructure at the core - to build a tiny lab that can be exploited to implement any new idea in something substantial. 
 
 ## Data
 
-- edge data, produced by microcontrollers-sensors and gateways, the atomic units of information representing state and values in a single, memorable point in time
-- data in motion, or fast data - seen as streams of certified information directed to points of first analysis and aggregation, useful to detect anomalies or state change before reaching any destination appointed to data-wrangling activities
-- data at rest, seen as data residing in a structured or semi-structured repository ready for full-blown analytics and phenomena discovery
+- __Edge data__, produced by microcontrollers-sensors and gateways, the atomic units of information representing state and values in a single, memorable point in time
+- __Data in motion__, or fast data - seen as streams of certified information directed to points of first analysis and aggregation, useful to detect anomalies or state change before reaching any destination appointed to data-wrangling activities
+- __Data at rest__, seen as data residing in a structured or semi-structured repository ready for full-blown analytics and phenomena discovery
 
-Let's get rid of the generic and overrated 'Data is the new oil' catchphrase for a moment, we just need to mention the types of data - and their velocity - that a model can efficiently process in each of the stages involved in building a development lab:
+Let's get rid of the generic and overrated "Data is the new oil" catchphrase for a moment, we just need to mention the types of data - and their velocity - that a model can efficiently process in each of the stages involved in building a development lab:
```

Comment:
I'm open to debate about the use of single quotes outside of double quotes. I've always defaulted to double quotes unless it's already in a quotation, but I'm sure there's some APA rule I've forgotten about.